src/train_cmd.cpp

```diff
@@ -2318,13 +2318,16 @@ static const byte _initial_tile_subcoord[6][4][3] = {
  * @param[out] path_found Whether a path has been found or not.
  * @param do_track_reservation Path reservation is requested
  * @param[out] dest State and destination of the requested path
+ * @param[out] final_dest Final tile of the best path found
```

Comment:
maybe a note to say this is only used for any depot orders?

Reply:
I would prefer to keep it as is because I can see other uses for it, for instance to show which platform of a station or which tile of a waypoint a train is heading to.